### PR TITLE
feat(widget-v2): skeleton for wallets page

### PR DESCRIPTION
**Pull request by samobasquiat**

### CODE REVIEW

**Review by RanGojo — APPROVED** [review]

File: widget/embedded/src/components/WalletModal/WalletModal.types.ts [hunk 1]
```diff
@@ -3,9 +3,9 @@ import type { WalletState } from '@rango-dev/ui';
 export interface PropTypes {
   open: boolean;
   onClose: () => void;
-  image: string;
-  state: WalletState;
-  error?: boolean;
+  image?: string;
+  state?: WalletState;
```
Comment (yeager-eren) [inline widget/embedded/src/components/WalletModal/WalletModal.types.ts:7]: It should alway has a state, why did you make it optional?

File: widget/ui/src/components/Wallet/Wallet.types.ts [hunk 2]
```diff
@@ -55,11 +55,11 @@ interface WalletProps {
   onClick: (type: WalletType) => void;
   selected?: boolean;
   description?: string;
+  loadingStatus?: 'failed' | 'success' | 'loading';
```
Comment (yeager-eren) [inline widget/ui/src/components/Wallet/Wallet.types.ts:58]: I think `status` is a better name here and also we can use `connected` instead of `sucess`.

Comment (samobasquiat) [reply widget/ui/src/components/Wallet/Wallet.types.ts:58]: It's unrelated to the wallet state. it's just for when Meta has not been loaded yet. I've modified both its name and type.

File: widget/embedded/src/components/WalletModal/WalletModal.types.ts (hunk 1 above)
Comment (samobasquiat) [reply widget/embedded/src/components/WalletModal/WalletModal.types.ts:7]: fixed.

**Pull request merged**